doc/mgr/dashboard.rst

```diff
@@ -314,12 +314,18 @@ The default value is 45 seconds.
 Enabling iSCSI Management
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Ceph Dashboard can manage iSCSI targets using the REST API provided
-by the `rbd-target-api` service of the `ceph-iscsi <https://github.com/ceph/ceph-iscsi>`_
-project. Please make sure that it's installed and enabled on the iSCSI gateways.
+The Ceph Dashboard can manage iSCSI targets using the REST API provided by the
+`rbd-target-api` service of the :ref:`ceph-isci`. Please make sure that it's
+installed and enabled on the iSCSI gateways.
+
+..note::
```

Comment:
This note section didn't render properly

Reply:
Good spotting, thanks! Fixed.